core/ledger/util/couchdb/metrics_test.go

```diff
@@ -1,46 +0,0 @@
-/*
```

Comment:
Did you delete this test file by mistake?

Reply:
Fixed.